lassen/sim.py

```diff
@@ -213,6 +213,7 @@ def __init__(self):
             self.rege: BitReg = BitReg()
             self.regf: BitReg = BitReg()
 
+        @name_outputs(res=Data, res_p=Bit, res_pp=Bit)
```

Comment:
Not sure if these names are right, and I completely made up `res_pp`.

@alexcarsello @nikhilbhagdikar what should we name these?

Reply:
I think the third one is `irq` (based on the return value)